assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties

```diff
@@ -509,10 +506,10 @@ che.infra.kubernetes.client.http.async_requests.max=1000
 # Number of maximum concurrent asynchronous web requests per host.
 che.infra.kubernetes.client.http.async_requests.max_per_host=1000
 
-# Max number of idle connections in the connection pool of the Kubernetes-client shared HTTP client.
+# Max number of idle connections in the connection pool of the `KubernetesClient` shared HTTP client.
```

Comment:
Is `KubernetesClient` a specific objects? I though we're talking about generic Kubernetes client here

Reply:
A few line before we have `KubernetesClient`. This change is for consistency.